onyo/commands/get.py

```diff
@@ -84,21 +85,21 @@
         action='store_true',
         default=False,
         help="""
-            Sort output in descending order (excludes --sort-ascending).
+            Sort output in descending order (excludes ``--sort-ascending``).
         """
     ),
 }
 
 
 def get(args: argparse.Namespace) -> None:
     """
-    Return values of the requested KEYs for matching assets.
+    Return values of the requested **KEY**\s for matching assets.
 
-    If no KEYs are given, all keys in the asset name are used (see
-    ``onyo.assets.filename``). If no PATHs are given, the current working
+    If no **KEY**\s are given, all keys in the asset name are printed (see
+    ``onyo.assets.filename``). If no **PATH**\s are given, the current working
     directory is used.
 
-    In addition to keys in asset contents, PSEUDO-KEYS can be queried and
+    In addition to keys in asset contents, **PSEUDO-KEYS** can be queried and
```

Comment:
Here it says `**PSEUDO-KEYS**` capitalized and bold-highlighted.
In the help-text for `'-M', '--match'` it is not highlighted this way.